src/dealDashboard/dealDiscussion/discussionThread/discussionThread.ts

```diff
@@ -226,17 +224,19 @@ export class DiscussionThread {
 
       // Early return, if there are no comments/discussions.
       if (!this.threadComments) return;
+      if (!this.dealDiscussion) {
+        if (this.discussionsService.discussions[discussionId]) {
+          this.dealDiscussion = this.discussionsService.discussions[discussionId];
```

Comment:
hydrating `this.dealDiscussion` was not working in the first place.
This fixes that.

credits @giladt